### PR TITLE
Adding multi lingual projects page.

### PR DESCRIPTION
**Pull request by gled4er**
Hello @timjpearson,

Please review the content we assembled for running multi-language projects.

Thank you!

### CODE REVIEW

**Comment by fnocera** [comment]
@gled4er please could you rebase and update your PR as the playbook has undergone a facelift and some of the structure has changed. Please reach out if it is not clear where this should live but perhaps within Team Agreements could be good https://github.com/microsoft/code-with-engineering-playbook/blob/fixing-url-source-control/team-agreements/readme.md

File: Engineering/MultiLingualProjects.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Multilingual Projects
```
Comment (davidkpiano) [inline Engineering/MultiLingualProjects.md]: Concern: "Multilingual" isn't a common way of expressing more than one language in programming. For ease in searching for related resources, could we consider using "polyglot" instead?

Comment (davidkpiano) [reply Engineering/MultiLingualProjects.md]: Reference: https://en.wikipedia.org/wiki/Polyglot_(computing)

**Comment by fnocera** [comment]
Please could you update and open a new PR for this given the new playbook structure? Thank you.

**Pull request closed without merging**